static/app/views/dashboardsV2/widgetBuilder/buildSteps/columnsStep/index.tsx

```diff
@@ -33,6 +34,7 @@ export function ColumnsStep({
   explodedFields,
   tags,
 }: Props) {
+  const customMeasurements = useCustomMeasurements().customMeasurements;
```

Comment:
nit: would this work?
```suggestion
  const {customMeasurements} = useCustomMeasurements();
```